force-app/main/default/lwc/debugLightningCmp/debugLightningCmp.js

```diff
@@ -0,0 +1,56 @@
+/************************************************************************ 
+(c) Copyright 2023 Avenoir Technologies Pvt. Ltd. All rights reserved.* 
+Created by: Santosh Kumar
+Ticket Number: AVEBLOG68
+------------------------------------------------------------------------* 
+Blog: How to debug standard lightning components.
+------------------------------------------------------------------------* 
+Version History:* 
+VERSION    DEVELOPER NAME        DATE         DETAIL FEATURES   
+1.0        Santosh Kumar        25/09/23     Initial Development   
+***********************************************************************/
+
+import { LightningElement,wire } from 'lwc';
+import { getRecord } from "lightning/uiRecordApi"
+
+const FIELDS = ["Contact.Name", "Contact.Phone","contact.Email"];
+
+export default class DebugLightningCmp extends LightningElement {
+
+    recordId = "003H4000001jJqiIAE";
```

Comment:
Do not hardcode Id use @api

Reply:
This is just for testing purpose and how we can debug component I am not using any parent child